modules/web-console/e2e/testcafe/components/Table.js

```diff
@@ -16,12 +16,13 @@
 
 import {Selector, t} from 'testcafe';
 
-const findCell = Selector((table, rowIndex, columnLabel) => {
-    const _findRowElement = (table, containerSelector) => {
```

Comment:
I'd add a type annotation for `columnId` to match that of `findCell` method.

Reply:
Added type descriptions.